chunks/lang-java/chunk.yaml

```diff
@@ -1,7 +1,7 @@
 variants:
   - name: "11"
     args:
-      JAVA_VERSION: 11.0.13
+      JAVA_VERSION: 11.0.15.fx-zulu
```

Comment:
What's the reasoning behind the zulu (Azul) version?

Reply:
It's the same as before (my PR didn't change that).

But, for historical context, I know that we looked at what `sdk list java` proposed, and we tried to select a "reasonable-looking" version (with FX enabled, because some Gitpod users need that).